src/DiffViewerProvider.ts

```diff
@@ -87,7 +74,7 @@ export class DiffViewerProvider implements vscode.CustomReadonlyEditorProvider<D
     return vscode.Uri.joinPath(this.context.extensionUri, "static", filename);
   }
 
-  private extractConfig(): Diff2HtmlConfig {
+  private static extractConfig(): Diff2HtmlConfig {
```

Comment:
This change is not necessary if `updateWebview` is not static.

Reply:
Correct, either neither is static, or both can be. You prefer them not to be static?

Reply:
Yep, not static.

Reply:
Got it, committed and pushed. Thanks for the guidance. Let me know if you'd like any further changes, I'll be happy if the new functionalities make it in. 8-)

Reply:
Thanks 😃